src/main/java/net/mcreator/ui/dialogs/tools/PackMakerToolUtils.java

```diff
@@ -46,4 +48,14 @@ public static void addGeneratableElementToWorkspace(Workspace workspace, FolderE
 		}
 	}
 
+	public static void addTagEntries(Workspace workspace, TagType tagType, String tagName, String... entries) {
+		TagElement tag = new TagElement(tagType, tagName);
+		if (!workspace.getTagElements().containsKey(tag)) {
+			workspace.addTagElement(tag);
+		}
+		for (String entry : entries) {
+			workspace.getTagElements().get(tag).add(entry);

```

Comment:
Entries are ArrayList, so manual duplicate check should be done here. For both managed and unmanaged entries